src/app/f2f/controllers/results.js

```diff
@@ -0,0 +1,71 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const { PROXY_API } = require("../../../../src/lib/config");
+
+class PostcodeSearchController extends BaseController {
+
+  locals(req, res, callback) {
+    super.locals(req, res, async (err, locals) => {
+      if (err) {
+        return callback(err, locals);
+      }
+
+      const userPostcode = req.sessionModel.get("postcode");
+      const resp = await req.axios.post(`https://${PROXY_API.BASE_URL}${PROXY_API.PATHS.POST_OFFICE}`, {
+        "searchString": userPostcode,
+        "productFilter": ["50321"]
+      });
+
+      locals.postcode = userPostcode;
+      locals.branch =
+      {
+        id: "branches",
+        name: "",
+        label: "hello",
```

Comment:
The label is now "", the `id: "branches"` has to stay (just in case you were wanting this removed too) as it's needed for the validation for the radio buttons